README.md

```diff
@@ -1,3 +1,5 @@
 # rocket
 
+# Watch scss command
```

Comment:
Name it more generic. Tomorrow, there'll be more parts to "watch" than just scss.